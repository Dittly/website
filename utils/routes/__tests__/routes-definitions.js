/* eslint-env jest */
import routesDefinitions from '../routes-definitions';

describe('utils/routes/routes-definitions', () => {
  it('contains all routes', () => {
    // This makes sure adding, changing or removing a route
    // doesn't happen accidentally.
    expect(routesDefinitions).toMatchSnapshot();
  });
});
