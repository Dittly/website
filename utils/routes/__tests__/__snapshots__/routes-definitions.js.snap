// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils/routes/routes-definitions contains all routes 1`] = `
Object {
  "authRoutes": Object {
    "login": Object {
      "name": "login",
      "page": "auth/login",
      "pattern": "/login",
    },
    "signup": Object {
      "name": "signup",
      "page": "auth/sign-up",
      "pattern": "/sign-up",
    },
  },
  "homeRoutes": Object {
    "home": Object {
      "name": "index",
      "pattern": "/",
    },
    "welcome": Object {
      "name": "welcome",
      "pattern": "/welcome",
    },
  },
  "projectRoutes": Object {
    "create": Object {
      "name": "projects/create",
      "pattern": "/projects/create",
    },
    "details": Object {
      "name": "projects/details",
      "pattern": "/projects/:id",
    },
  },
}
`;
